tools/mksurfdata_map/unit_testers/test_mkindexmapMod.F90

```diff
@@ -200,111 +202,6 @@ subroutine test_get_dominant_indices
    end subroutine test_get_dominant_indices
 !------------------------------------------------------------------------------
 
-!------------------------------------------------------------------------------
```

Comment:
@slevisconsulting thanks for working on this. The changes above are the ones that I expected. Below here you are deleting the general test "test_filter_same", which is the part I wondered about.  Shouldn't this test just be changed to use